helm/values-prod.yaml

```
@@ -0,0 +1,181 @@
+      TRANSACTIONS_NOTIFICATIONS_QUEUE_NAME: pagopa-p-weu-ecommerce-transaction-notifications-queue
+      PAYMENT_TRANSACTIONS_GATEWAY_URI: https://api.platform.pagopa.it/payment-transactions-gateway/internal/v1
+      PAYMENT_TRANSACTIONS_GATEWAY_READ_TIMEOUT: "10000"
+      PAYMENT_TRANSACTIONS_GATEWAY_CONNECTION_TIMEOUT: "10000"
+      NODO_URI: https://api.platform.pagopa.it/nodo/nodo-per-pm/v2
+      NODO_READ_TIMEOUT: "10000"
+      NODO_CONNECTION_TIMEOUT: "10000"
+      MONGO_HOST: pagopa-p-weu-ecommerce-cosmos-account.mongo.cosmos.azure.com
+      MONGO_USERNAME: pagopa-p-weu-ecommerce-cosmos-account
+      MONGO_PORT: "10255"
+      MONGO_SSL_ENABLED: "true"
+      ECOMMERCE_STORAGE_QUEUE_ACCOUNT_NAME: pagopapweuecommercesa
+      ECOMMERCE_STORAGE_QUEUE_ENDPOINT: https://pagopapweuecommercesa.queue.core.windows.net/pagopauweuecommercesa
+      REFUND_RETRY_EVENT_BASE_INTERVAL_SECONDS: "600"
+      REFUND_RETRY_EVENT_MAX_ATTEMPTS: "3"
+      CLOSE_PAYMENT_RETRY_EVENT_BASE_INTERVAL_SECONDS: "600"
+      CLOSE_PAYMENT_RETRY_EVENT_MAX_ATTEMPTS: "3"
+      NOTIFICATION_RETRY_EVENT_BASE_INTERVAL_SECONDS: "600"
+      NOTIFICATION_RETRY_EVENT_MAX_ATTEMPTS: "3"
+      NOTIFICATIONS_SERVICE_URI: https://api.platform.pagopa.it/ecommerce/notifications-service/v1
+      NOTIFICATIONS_SERVICE_READ_TIMEOUT: "10000"
+      NOTIFICATIONS_SERVICE_CONNECTION_TIMEOUT: "10000"
+      PERSONAL_DATA_VAULT_API_BASE_PATH: "https://api.tokenizer.pdv.pagopa.it/tokenizer/v1"
+      ELASTIC_APM_SERVICE_NAME: pagopa-ecommerce-event-dispatcher-service-blue
+      ELASTIC_APM_APPLICATION_PACKAGES: it.pagopa.ecommerce.eventdispatcher
+      ELASTIC_APM_SERVER_URL: http://quickstart-apm-http.elastic-system.svc:8200
+      ELASTIC_APM_LOG_SENDING: "true"
+    envSecret:
+      ECOMMERCE_STORAGE_QUEUE_KEY: ecommerce-storage-account-key
+      MONGO_PASSWORD: mongo-ecommerce-password
+      NODO_CONNECTION_STRING: nodo-connection-string
+      QUEUE_CONNECTION_STRING: ecommerce-storage-connection-string
+      NOTIFICATIONS_SERVICE_API_KEY: notifications-service-api-key
+      PERSONAL_DATA_VAULT_API_KEY: personal-data-vault-api-key
+      ELASTIC_APM_SECRET_TOKEN: elastic-apm-secret-token
+      PAYMENT_TRANSACTIONS_GATEWAY_API_KEY: payment-transactions-gateway-api-key
+  image:
+    repository: pagopapcommonacr.azurecr.io/pagopaecommerceeventdispatcherservice
+    tag: "0.4.12"
+    pullPolicy: Always
+  livenessProbe:
+    httpGet:
+      path: /actuator/health/liveness
+      port: 9000
+    initialDelaySeconds: 60
+    failureThreshold: 6
+    periodSeconds: 10
+  readinessProbe:
+    httpGet:
+      path: /actuator/health/readiness
+      port: 9000
+    initialDelaySeconds: 60
+    failureThreshold: 6
+    periodSeconds: 10
+  service:
+    create: true
+    type: ClusterIP
+    ports:
+      - 9000
+  ingress:
+    create: true
+    host: "weuuat.ecommerce.internal.uat.platform.pagopa.it"
```

This should be the 
```suggestion
    host: "weuprod.ecommerce.internal.platform.pagopa.it"
```
Right?